showcase/scripts/generate_showcase.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Generates showcase without post processing (i.e. raw gapic, grpc and proto
+# libraries). It will compute the showcase version from
+# `sdk-platform-java/showcase/gapic-showcase/pom.xml`. The generator version is
+# inferred from versions.txt
+set -ex
+
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+lib_gen_scripts_dir=$script_dir/../../library_generation/
+source $lib_gen_scripts_dir/utilities.sh
+
+cd $script_dir
+
+# clone gapic-showcase
+if [ ! -d schema ]; then
+  if [ -d gapic-showcase ]; then
+    rm -rdf gapic-showcase
+  fi
+  showcase_version=$(xmllint --xpath '/*[local-name()="project"]/*[local-name()="properties"]/*[local-name()="gapic-showcase.version"]/text()' $script_dir/../gapic-showcase/pom.xml)
```

Comment:
Can you add a comment explaining where this showcase version is being extracted form?

Reply:
Added